packages/turbo-gen/src/utils/plop.ts

```diff
@@ -139,29 +140,26 @@ export function getCustomGenerators({
       // turbo
       parts.pop();
       const workspaceRoot = path.join("/", ...parts);
-      return workspaceRoot == w.paths.root;
+      return workspaceRoot === w.paths.root;
     });
 
     if (gensWorkspace) {
-      if (!gensByWorkspace[gensWorkspace.name]) {
+      if (!(gensWorkspace.name in gensByWorkspace)) {
         gensByWorkspace[gensWorkspace.name] = [];
       }
       gensByWorkspace[gensWorkspace.name].push(generatorDetails);
     } else {
-      if (!gensByWorkspace["root"]) {
-        gensByWorkspace["root"] = [];
+      if (!("root" in gensByWorkspace)) {
+        gensByWorkspace.root = [];
       }
-      gensByWorkspace["root"].push(generatorDetails);
+      gensByWorkspace.root.push(generatorDetails);
     }
   });
 
   // add in separators to group by workspace
-  const gensWithSeparators: Array<Generator | inquirer.Separator> = [];
-  const lastGroup = undefined;
+  const gensWithSeparators: Array<Generator | Separator> = [];
   Object.keys(gensByWorkspace).forEach((group) => {
-    if (group !== lastGroup) {
-      gensWithSeparators.push(new inquirer.Separator(group));
-    }
+    gensWithSeparators.push(new Separator(group));
```

Comment:
Just confirming: the removed conditional was just always true, correct?

Reply:
That's correct, this was a bad if that ts linting pointed out